test/test_aqua.jl

```diff
@@ -1,7 +1,9 @@
 @testitem "Aqua" tags=[:aqua] begin
     using Aqua
+    import QuantumInterface as QI
+     own_types = [QI.AbstractBra, QI.AbstractKet, QI.AbstractSuperOperator, QI.AbstractOperator]
```

Comment:
Could you elaborate why these need to be marked as `own`? I am not sure whether this would be consistent with how independent packages downstream of QuantumInterface (e.g. QuantumOptics or QuantumCummulants) -- I want to make sure there are some precautions put in place for potential future issues.